repository/src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -181,11 +191,27 @@ public ContentWriter getWriter(NodeRef nodeRef, QName propertyQName, boolean upd
      * @return A direct access {@code URL} object for the content.
      * @throws UnsupportedOperationException if the store is unable to provide the information.
      */
+    @Deprecated
     default DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachment)
     {
         return requestContentDirectUrl(nodeRef, attachment, null);
     }
 
+    /**
+     * Gets a presigned URL to directly access the content. It is up to the actual store
+     * implementation if it can fulfil this request with an expiry time or not.
+     *
+     * @param nodeRef Node ref for which to obtain the direct access {@code URL}.
+     * @param attachment {@code true} if an attachment URL is requested, {@code false} for an embedded {@code URL}.
+     * @param propertyQName the name of the property, which must be of type <b>content</b>
```

Comment:
Trivial: re-order in JavaDoc to match order in method sig

Reply:
Resolved. See https://github.com/Alfresco/alfresco-community-repo/pull/813/commits/ca25e3929b24c9fb3bbb5ff89190930839a09a04